agent/php_user_instrument.c

```diff
@@ -58,9 +58,7 @@ int nr_zend_call_orig_execute(NR_EXECUTE_PROTO TSRMLS_DC) {
     NR_PHP_PROCESS_GLOBALS(orig_execute)
     (NR_EXECUTE_ORIG_ARGS_OVERWRITE TSRMLS_CC);
   }
-  zend_catch {
-    zcaught = 1;
-  }
+  zend_catch { zcaught = 1; }
```

Comment:
Is this the result of clang-formatting? 
My clang-format sets this line to the original format, which uses the .clang-format file at the top level directory.

Reply:
Yes, we can revert this change, not sure why clang-format changed it.